### PR TITLE
Quoted names

### PR DESCRIPTION
**Pull request by Claude-at-Instaclustr**
Added quoted name parsing,
Changed version from 0.1.1 to 0.2.0

- [ ] Requires tree-sitter-cql3 parser v 0.1.0

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -24,7 +24,7 @@ serde = { version = "1.0.111", features = ["derive"] }
 
 # Parsers
 tree-sitter = "0.20.5"
-tree-sitter-cql = "0.0.1"
+tree-sitter-cql = { git = "https://github.com/shotover/tree-sitter-cql.git", branch="main"  }
```
Comment (Claude-at-Instaclustr) [inline Cargo.toml]: - [ ] #This like must be changed when tree-sitter-cql is released.  This pull depends upon it.

**Review by Claude-at-Instaclustr — COMMENTED** [review]
changes required

**Comment by rukai** [comment]
I suspect this might be your last PR?
I wont be able to review till monday but thankyou for this!

**Review by rukai — APPROVED** [review]
The new API and implementation LGTM.
I've pushed a commit to use tree-sitter-cql 0.1.0 so that we can merge this PR.

**Pull request merged**